service-tests/github.js

```diff
@@ -66,6 +66,176 @@ t.create('GitHub closed issues raw')
     value: Joi.string().regex(/^\w+\+?$/)
   }));
 
+// pull request state
+
+t.create('Pull request state, regular case')
+  .get('/pr-state/badges/shields/578.json')
```

Comment:
Let's make this `/pulls/state` for consistency with e.g. `https://img.shields.io/github/pulls/detail/comments/badges/shields/1112.svg`.